src/goto-instrument/contracts/contracts.cpp

```diff
@@ -15,7 +15,10 @@ Date: February 2016
 
 #include <algorithm>
 #include <map>
+#include <util/find_symbols.h>
+#include <util/std_code.h>
```

Comment:
Nit picking: please move the `util` includes a few lines down, where the other `util` includes are.

Reply:
done